lectures/inequality.md

```diff
@@ -280,44 +328,49 @@ Lorenz curve.
 
 ### Definition
 
-
 As before, suppose that the sample $w_1, \ldots, w_n$ has been sorted from
 smallest to largest.
 
 The Gini coefficient is defined for the sample above as 
 
-\begin{equation}
-    \label{eq:gini}
-    G :=
-    \frac
-        {\sum_{i=1}^n \sum_{j = 1}^n |w_j - w_i|}
-        {2n\sum_{i=1}^n w_i}.
-\end{equation}
+$$
+G :=
+\frac{\sum_{i=1}^n \sum_{j = 1}^n |w_j - w_i|}
+     {2n\sum_{i=1}^n w_i}.
+$$ (eq:gini)
 
 
 The Gini coefficient is closely related to the Lorenz curve.
 
 In fact, it can be shown that its value is twice the area between the line of
 equality and the Lorenz curve (e.g., the shaded area in the following Figure below).
 
+```{note}
+Another way to think of the gini coefficient is the area between the 45-degree line of 
+perfect equality and the Lorenz curve minus the area below the Lorenz curve devided by
+the total area below the 45-degree line. 
```

Comment:
Is this correct?  It sounds surprising.  Could we please have a reference?

Reply:
thanks @jstac I have clarified in [de796df](https://github.com/QuantEcon/lecture-python-intro/pull/384/commits/de796df498bd29fb8fd7440341a4ed66f2806ca9)